includes/overlords/contribs.php

```diff
@@ -301,6 +313,44 @@ public static function display_contribs($mode, $hierarchy_ids, $branch = false,
 				);
 			break;
 
+			case 'featured':
+				$is_featured = true;
+
+				// Get a list of all valid contrib_ids and put them in an array
+				$sql_ary = [
+					'SELECT' => 'c.contrib_id',
+
+					'FROM' => [
+						TITANIA_REVISIONS_TABLE => 'r',
+						TITANIA_CONTRIBS_TABLE => 'c',
+					],
+
+					'WHERE'	=> 'r.revision_status = ' . ext::TITANIA_REVISION_APPROVED . '
+						AND r.revision_submitted = 1
+						AND c.contrib_status = ' . ext::TITANIA_CONTRIB_APPROVED . '
+						AND r.contrib_id = c.contrib_id',
+
+					'ORDER_BY'	=> 'c.contrib_id ASC',
+				];
+
+				$sql = phpbb::$db->sql_build_query('SELECT_DISTINCT', $sql_ary);
```

Comment:
Will this include MODs which we aren't displaying anymore?

Reply:
That's a good point. I might also see if we can add some extra filtering on it so it will only pull from the latest phpBB version, so we don't end up with something like a 3.0 style making the featured list.